modules/tide_media/tide_media.info.yml

```diff
@@ -23,3 +23,4 @@ dependencies:
   - svg_image:svg_image
   - video_embed_field:video_embed_field
   - video_embed_field:video_embed_media
+  - media_entity_file_replace
```

Comment:
hey @sharmasahil , I think the format should be `media_entity_file_replace:media_entity_file_replace`